homeassistant/components/apple_tv/__init__.py

```diff
@@ -337,7 +337,8 @@ async def _connect(self, conf):
         self._connection_attempts = 0
         if self._connection_was_lost:
             _LOGGER.info(
-                'Connection was re-established to Apple TV "%s"', self.atv.service.name
+                'Connection was re-established to Apple TV "%s"',
+                self.config_entry.data.get(CONF_NAME),
```

Comment:
Isn't `CONF_NAME` always available from `config_entry.data`?

Reply:
You are right, there are a few instances of doing this that I fixed now.